samples/Getting_started.ipynb

```diff
@@ -38,9 +38,13 @@
     }
    },
    "source": [
-    "### Authenticating with your Data Portal log in\n",
+    "### Authenticating with Cirro\n",
     "\n",
-    "Before you use the Cirro CLI, you will need to authenticate yourself. You can do this by running the command below. If the package cannot already find your information (i.e. you or your machine do not have credentials already saved), then this command it will give you a link to log in through a browser window, the same way you would log into the Data Portal website."
+    "Before you use the Cirro CLI, you will need to authenticate yourself. You can do this by running the command below. If the package cannot already find your information (i.e., you do not have credentials already saved), then this command will give you a link to log in through a browser window, the same way you would log into the Data Portal website.\n",
+    "\n",
+    "You will also need to set the base URL of the Cirro instance you wish to access. To find it, log in to Cirro and check the URL in your browser.\n",
```

Comment:
Could we put the example of 'app.cirro.bio' in this line too? I know its in a lot of places, I just want to be clear what they're looking for when we say "check the url"